cmd/cron_tasks.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"github.com/Scalingo/cli/appdetect"
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/crontasks"
+	"github.com/urfave/cli"
+)
+
+var (
+	CronTasksCommand = cli.Command{
+		Name:     "cron-jobs",
+		Category: "Cron Jobs",
+		Flags:    []cli.Flag{appFlag},
+		Usage:    "List the cron jobs of an application",
+		Description: `List all the cron jobs of an application:
+
+    $ scalingo --app my-app cron-jobs`,
+
+		Action: func(c *cli.Context) {
+			currentApp := appdetect.CurrentApp(c)
+			var err error
+			if len(c.Args()) == 0 {
+				err = crontasks.List(currentApp)
+			} else {
+				cli.ShowCommandHelp(c, "cron-jobs")
+			}
+
+			if err != nil {
+				errorQuit(err)
+			}
```

Comment:
You should leave the function as early as possible in case of problem

```suggestion
			if len(c.Args()) > 0 {
				cli.ShowCommandHelp(c, "cron-jobs")
				return
			}

			currentApp := appdetect.CurrentApp(c)			
			err  := crontasks.List(currentApp)
			if err != nil {
				errorQuit(err)
			}
```